ui/src/components/Rules/Rule/RuleCreator/CommunityRuleUpload/index.tsx

```diff
@@ -24,6 +25,7 @@ const CommunityRuleUpload = (props: ICommunityRuleUpload) => {
         type: props.type,
         description: descriptionRef.current.value,
         JsonRules: props.rules,
+        uploadedBy: uploadedByRef.current.value || undefined,
       })
         .then((resp) => {
           if (resp.code === 1) {
```

Comment:
You could save `resp.result` here and use that instead of the generic message when an upload fails; _Something went wrong uploading your rules. Please try again later_